### PR TITLE
Added yarn gulp debug-no-start command

### PR DESCRIPTION
**Pull request by limonspb**
` yarn gulp debug-no-start` command allows building debug version of the Configurator without starting it afterward. Makes it handy to debug Configurator directly in VS code.

For debugging under Windows 10:
(to be added in wiki after merging this PR)
(credits to Pierre Gazzano for the initial launch.json text and idea)

create launch.json file in .vscode folder with the following content:
```
{
    "version": "0.2.0",
    "configurations": [
        {
            "name": "Launch Betaflight",
            "type": "nwjs",
            "request": "launch",
            "runtimeExecutable": "${workspaceRoot}/debug/betaflight-configurator/win64/betaflight-configurator.exe",
            "runtimeArgs": [
                "${workspaceRoot}",
                "--remote-debugging-port=9222"
            ],
            "port": 9222,
            "nwjsVersion": "any",
            "webRoot":"${workspaceRoot}/src",
            "preLaunchTask": "build and copy package.json",
            "postDebugTask": "delete package.json"
        },
        {
            "type": "nwjs",
            "request": "attach",
            "name": "Attach to Betaflight",
            "port": 9222,
            "webRoot":"${workspaceRoot}/src/js",
            "verbose":true,
            "reloadAfterAttached": true
        }
    ]
}
```

create tasks.json file in .vscode folder with the following content:
```
{
    "version": "2.0.0",
    "tasks": [
        {
            "label": "delete package.json",
            "type": "shell",
            "command": "rm .\\src\\package.json",
            "problemMatcher": [],
            "group": {
                "kind": "build",
                "isDefault": true
            },
            "presentation": {
                "showReuseMessage": false
            }
        },
        {
            "label": "build debug no start",
            "type": "shell",
            "command": "yarn gulp debug-no-start; copy -Path package.json -Destination .\\src\\package.json -Force",
            "problemMatcher": [],
            "group": {
                "kind": "build",
                "isDefault": true
            },
            "presentation": {
                "showReuseMessage": false
            }
        },
        {
            "label": "copy package.json",
            "type": "shell",
            "command": "copy -Path package.json -Destination .\\src\\package.json -Force",
            "problemMatcher": [],
            "group": {
                "kind": "build",
                "isDefault": true
            },
            "presentation": {
                "showReuseMessage": false
            }
        },
        {
            "label": "build and copy package.json",
            "dependsOn": [
                "build debug no start",
                "copy package.json"
            ],
            "presentation": {
                "showReuseMessage": false
            }
        }
    ]
}
```
Press F5 in VS code - it will build Configurator and launch it. Now can set breakpoints directly in the VS code.

### CODE REVIEW

**Review by chmelevskij — APPROVED** [review]

**Review by haslinghuis — APPROVED** [review]

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> FAIL
 - cooling off period lapsed -> FAIL
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> FAIL
 - `Tested` label found -> FAIL
 - assigned to an approver -> FAIL
 - approver count at least three -> FAIL

**Comment by McGiverGim** [comment]
What about store this information at some place to not loose it? Maybe a `doc/vscode` folder in the repo containing a `md` file with this information and the `json` files? I think is the easier place. Thoughts?

**Comment by chmelevskij** [comment]
Or we can put it in the repo. The only thing I'm not sure about is how to deal with path separators between OSes. I think vscode would understand them in any format, but not 100% sure.

Screenshot 2021-09-16 175138

**Comment by haslinghuis** [comment]
Agree with @McGiverGim here.

**Comment by limonspb** [comment]
> What about store this information at some place to not loose it? Maybe a `doc/vscode` folder in the repo containing a `md` file with this information and the `json` files? I think is the easier place. Thoughts?
> 
> EDIT: I see you plan to add it to the wiki. The Configurator has no wiki. Are you talking about firmware repo?

Pierre has added some information here aleady:
https://github.com/betaflight/betaflight/wiki/How-to-debug-Betaflight-configurator-with-Visual-Studio-Code

I think after this PR is merged we can just update that wiki? Or still `doc/vscode` would be a better solution?

**PS: and what is happening with Android builds?**

**Comment by haslinghuis** [comment]
We could let the info in the wiki mature to become adopted later in docs. It should not hold up this PR as the docs are in the firmware repo anyway.

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> FAIL
 - cooling off period lapsed -> PASS
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> FAIL
 - `Tested` label found -> FAIL
 - assigned to an approver -> FAIL
 - approver count at least three -> FAIL

**Comment by McGiverGim** [comment]
> Or we can put it in the repo. The only thing I'm not sure about is how to deal with path separators between OSes. I think vscode would understand them in any format, but not 100% sure.
> 
> Screenshot 2021-09-16 175138

This is a good idea... if we are sure this files will be maintained and everybody will use this files as they are, without modifications, what is probably wrong.

For this reason maybe is better to have them in a folder, in this way everyone can do what they want in their IDE. But if this information is already in the Betaflight firmware wiki, to me is ok. Maybe not the best place, but at least it will not be lost.

**Review by McGiverGim — APPROVED** [review]

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> FAIL
 - cooling off period lapsed -> PASS
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> FAIL
 - `Tested` label found -> FAIL
 - assigned to an approver -> FAIL
 - approver count at least three -> PASS

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> PASS
 - cooling off period lapsed -> PASS
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> PASS
 - `Tested` label found -> FAIL
 - assigned to an approver -> PASS
 - approver count at least three -> PASS

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> PASS
 - assigned to a milestone -> PASS
 - cooling off period lapsed -> PASS
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> PASS
 - `Tested` label found -> FAIL
 - assigned to an approver -> PASS
 - approver count at least three -> PASS

**Comment by haslinghuis** [comment]
Tested ok on Linux.

**Comment by blckmn** [comment]
AUTOMERGE: (PASS)
 - github identifies PR as mergeable -> PASS
 - assigned to a milestone -> PASS
 - cooling off period lapsed -> PASS
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> PASS
 - `Tested` label found -> PASS
 - assigned to an approver -> PASS
 - approver count at least three -> PASS

**Comment by limonspb** [comment]
Wiki has been updated:
https://github.com/betaflight/betaflight/wiki/How-to-debug-Betaflight-configurator-with-Visual-Studio-Code

**Comment by haslinghuis** [comment]
Added instructions for Linux too. For OSX probably only need to change the platform to osx64 in launch.json on line 11

**Pull request merged**
